src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import SemanticDatepicker from './components/datepicker';
+
+export default SemanticDatepicker;
```

Comment:
you can write like this

`export { default }  from './components/datepicker';`